test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       serviceAccountName: csi-gce-pd-controller-sa
       containers:
         - name: csi-snapshotter
-          image: registry.k8s.io/sig-storage/csi-snapshotter:v8.2.0
+          image: registry.k8s.io/sig-storage/csi-snapshotter:v8.3.0
```

Comment:
question: we intentionally want to diverge from upstream here, or do we have plans to update upstream as well?

Reply:
Upstream fix https://github.com/kubernetes/kubernetes/pull/133117 megred ^^.